src/main/java/org/infoshareacademy/javaholics/AllRoute/CreateRoute.java

```diff
@@ -0,0 +1,96 @@
+package org.infoshareacademy.javaholics.AllRoute;
+import org.infoshareacademy.javaholics.Route;
+import org.infoshareacademy.javaholics.RouteDifficulty;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.util.Scanner;
+import java.util.Date;
+
+public class CreateRoute {
+    boolean status;
+    Date date = new Date();
+    Route route = new Route();
+    public Date getDate() {
+        return date;
+    }
+
+    Scanner scanner = new Scanner(System.in);
+    public void routeLength (){
+        System.out.println("Podaj początek trasy : ");
+        route.setPlaceStart(scanner.nextLine());
+        System.out.println("Podaj koniec trasy : ");
+        route.setPlaceStop(scanner.nextLine());
+        System.out.println("Podaj długość trasy w km : ");
+        route.setLength(scanner.nextInt());
+        System.out.println("Początek twojej trasy jest : " + route.getPlaceStart() + " , a koniec jest " + route.getPlaceStop());
+        System.out.println("Całkowita długość trasy to " + route.getLength());
+
+    }
+    public void routeLevelOfDifficulty (){
+        System.out.println("Podaj poziom trudności trasy :");
+        System.out.println("1 - Łatwy");
+        System.out.println("2 - Średni");
+        System.out.println("3 - Wymagający");
+        System.out.println("4 - Bardzo trudny");
+        int levelOfDifficulty = scanner.nextInt();
+        if (levelOfDifficulty == 1){
+            System.out.println("Poziom trasy - Łatwy");
+            route.setDifficulty(RouteDifficulty.EASY);
+        }
+        if (levelOfDifficulty == 2){
+            System.out.println("Poziom trasy - Średni");
+            route.setDifficulty(RouteDifficulty.MEDIUM);
+        }
+        if (levelOfDifficulty == 3){
+            System.out.println("Poziom trasy - Wymagający");
+            route.setDifficulty(RouteDifficulty.EXTREME);
+        }
+        if (levelOfDifficulty == 4){
+            System.out.println("Poziom trasy - Bardzo trudny");
+            route.setDifficulty(RouteDifficulty.HARD);
+        }
+        System.out.println("Proponowany poziom trudności stworzonej trasy to " + route.getDifficulty());
+    }
+    public void routeType (){
+        System.out.println("Podaj typ trasy :  ");
+        System.out.println("1 - Trasa spacerowa ");
+        System.out.println("2 - Trasa rowerowa ");
+        System.out.println("3 - Trasa trekingowa ");
+        int typeRoute = scanner.nextInt();
```

Comment:
wyątki i metoda wprowadzania powinna byc inna w zależnosci czy oczekujemy liczby czy tekstu - myśle ze powinny być to dwie metody 
dla getstringa zabezpieczenie przed za długim tekstem (poza wartość dozowlanoa stringa) z opisem ze opis poinien miec mniej niż x znaków, myśle ze w ogole powinniśmy ustalić jakiś limit, nazwy, miejscowosc itp bo jak dojdziemy do aplikacji weowej to nie bedziemy chcieli nazyw trasy na 3 zdania:) można użyć prostej funkcji sprawdzajacej . leght
dal getintegera wyjątki to minus, zero, string, etc, pytanie czy dla dłgości nie dpouścic ułamków